client/src/components/ElementEditor/Header.js

```diff
@@ -79,11 +78,10 @@ class Header extends Component {
     );
   }
 
-
   toggle() {
-    this.setState({
-      tooltipOpen: !this.state.tooltipOpen
-    });
+    this.setState((prevState) => ({
```

Comment:
As above

Reply:
See [this comment](https://github.com/silverstripe/silverstripe-elemental/pull/1070#discussion_r1253833874)